src/Sylius/Bundle/ImportExportBundle/Behat/ImportExportContext.php

```diff
@@ -0,0 +1,637 @@
+        }
+    }
+
+    /**
+     * @Then Export job for export profile :exportProfile should be created
+     */
+    public function exportJobForThisExportProfileShouldBeCreated($exportProfileName)
+    {
+        $this->profile = $this->getRepository('export_profile')->findOneBy(array('name' => $exportProfileName));
+        $this->filePath = $this->profile->getWriterConfiguration()['file'];
+        $this->job = $this->profile->getJobs()->current();
+        $this->type = 'export';
+
+        if (false === $this->job) {
+            throw new \Exception('Export job has not been created');
+        }
+    }
+
+    /**
+     * @Then I should be on its details page
+     */
+    public function iShouldBeOnItsDetailsPage()
+    {
+        if (null === $this->job || null === $this->profile) {
+            throw new \Exception('Job or profile was not set. Make "Export job for export profile " " should be created" before you use this step.');
+        }
+
+        $path = sprintf('sylius_backend_%s_job_show', $this->type);
+
+        $this->getSession()->visit($this->generatePageUrl($path,
+            array(
+                'profileId' => $this->profile->getId(),
+                'id' => $this->job->getId(),
+            )
+        ));
+    }
+
+    /**
+     * @Then Import job for import profile :importProfileName should be created
+     */
+    public function importJobForImportProfileShouldBeCreated($importProfileName)
+    {
+        $this->profile = $this->getRepository('import_profile')->findOneBy(array('name' => $importProfileName));
+        $this->filePath = $this->profile->getReaderConfiguration()['file'];
+        $this->job = $this->profile->getJobs()->current();
+        $this->type = 'import';
+
+        if (false === $this->job) {
+            throw new \Exception('Import job has not been created');
+        }
+    }
+
+    /**
+     * @param string    $name
+     * @param string    $description
+     * @param string    $code
+     * @param string    $reader
+     * @param string    $readerConfiguration
+     * @param string    $writer
+     * @param string    $writerConfiguration
+     * @param bool|true $flush
+     *
+     * @return ImportProfileInterface
+     */
+    private function thereIsImportProfile(
+        $name,
+        $description,
+        $code,
+        $reader,
+        $readerConfiguration,
+        $writer,
+        $writerConfiguration,
+        $flush = true
+    ) {
+        $repository = $this->getRepository('import_profile');
+
+        return $this->createProfileInDatabase($repository, $description, $code, $reader, $readerConfiguration, $writer, $writerConfiguration, $name, $flush);
+    }
+
+    /**
+     * @param string    $name
+     * @param string    $description
+     * @param string    $code
+     * @param string    $reader
+     * @param string    $readerConfiguration
+     * @param string    $writer
+     * @param string    $writerConfiguration
+     * @param bool|true $flush
```

Comment:
Same as above and below (won't write at all places, 'cause I got lags while watching this PR).